lib/AuthoritiesSearchForm/AuthoritiesSearchForm.js

```diff
@@ -193,12 +191,13 @@ const AuthoritiesSearchForm = ({
                 {intl.formatMessage({ id: 'stripes-authority-components.label.search' })}
               </Button>
             </div>
-            <Row between="xs">
-              <Col xs={12} lg={6}>
+            <div className={css.resetAndAdvancedSearchWrapper}>
+              <div className={css.resetButtonWrapper}>
```

Comment:
Have you tried to add styles via Button's prop `buttonClass`?

Reply:
Updated styles